react/week3/todos/src/TodosList.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import Border from "./Border";
+import PropTypes from "prop-types";
+
+function TodoItem({ todo, DeleteTodo, UpdateTodo }) {
+  const [isCheked, setIsCheked] = useState(false);
+  const [isUpdated, setIsUpdated] = useState(true);
+  const [todoDescription, settodoDescription] = useState(todo.description);
+  const updateDescription = (e) => {
+    settodoDescription(e.target.value);
+  };
+  return (
+    <li className={isCheked ? "Done" : "notDone"}>
```

Comment:
You are not using the `notDone` for currently, so you could use an empty string:
```suggestion
    <li className={isCheked ? "Done" : ""}>
```
or use `&&`